lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,10723 @@
+{
```

Comment:
<p>Budem tu chtít poznamenat, že alternativou je ještě df.copy(), nebo to si necháme na jindy?</p><p> </p><p>(Byla teď moc pěkná přednáška Vincenta Warmerdama, kde ukazoval pandas pipeliny a na začátku udělal vždy copy, aby měl proces bez side effectů).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>"Mimochodem, kopii tabulky můžeš kdykoliv vytvořit metodou&nbsp;<code><a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.copy.html" target="_blank">copy</a></code>&nbsp;- to se hodí třeba při psaní funkcí, kde se vstupní tabulka z různých důvodů upravuje."</p>

Reply:
jj, něco takovýho